Sources/KSCrashRecording/KSCrashReportC.c

```diff
@@ -1102,11 +1110,13 @@ static void writeAllThreads(const KSCrashReportWriter *const writer, const char
         KSLOG_DEBUG("Writing %d threads.", threadCount);
         for (int i = 0; i < threadCount; i++) {
             KSThread thread = ksmc_getThreadAtIndex(context, i);
+            int threadRunState = 0;
```

Comment:
I'd suggest giving the thread state value 0 a name here, like `#define TH_STATE_UNSET 0` in the header, so that we know that `threadRunState` will still contain an actionable value if `ksthread_getThreadState` fails.

Actually, it might be an idea to set it from within `ksthread_getThreadState` so that the invariant for values of this type always hold after calling it.

Reply:
Done